build.gradle

```diff
@@ -45,6 +45,8 @@ bootRepackage {
     enabled = false
 }
 
+project.ext.set("local", "")
```

Comment:
similar remark as in scheduling api, local should not be forced but passed as a build parameter -Plocal

Reply:
removed